app/assets/stylesheets/procedure_champs_editor.scss

```diff
@@ -52,6 +52,7 @@
     .head {
       select {
         margin-bottom: 0px;
+        font-size: smaller;
```

Comment:
```suggestion
        font-size: smaller;
```
pareil, je préfère ne pas toucher au style du DSFR pour des raisons d'homogénéité

Reply:
Voilà 
AVANT
![image](https://github.com/demarches-simplifiees/demarches-simplifiees.fr/assets/15379878/18e77a8c-8839-488f-a375-cd15475db861)
APRES 
![image](https://github.com/demarches-simplifiees/demarches-simplifiees.fr/assets/15379878/679e80b2-6aeb-47fa-baeb-c56cd19c4335)

Personnellement je pense qu'on manque qqc : le déplacement devient aussi important que le label du champ et son type. Ce qui ne veut pas dire que la manière de faire était bonne :-) mais il nous manque une idée pour que le design soit adapté. Pour l'instant je l'ai mis comme demandé.